app/src/main/java/org/mozilla/fenix/BrowserDirection.kt

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix
+
+import androidx.annotation.IdRes
+
+enum class BrowserDirection(@IdRes val fragmentId: Int?) {
+    FromGlobal(null),
```

Comment:
Proguard/R8 converts enums to ints if possible to avoid all the overhead of creating classes. I'm not sure if using a Nullable Int value will inhibit this optimization.